wakuv2/nwaku.go

```diff
@@ -2165,22 +2181,27 @@ func (w *Waku) AddRelayPeer(address multiaddr.Multiaddr) (peer.ID, error) {
 }
 
 func (w *Waku) DialPeer(address multiaddr.Multiaddr) error {
-	// TODO-nwaku
-	/*
-		ctx, cancel := context.WithTimeout(w.ctx, requestTimeout)
-		defer cancel()
-		return w.node.DialPeerWithMultiAddress(ctx, address) */
-	return nil
+	return w.WakuDialPeer(address, "", 1000)
```

Comment:
Instead of hardcoding the 1000, we can use instead `requestTimeout / time.Millisecond`

Reply:
ooh that looks much better!